data/dataset.py

```diff
@@ -91,24 +91,24 @@ def __getitem__(self, idx: int) -> Union[Data, HeteroData]:
             sampled_edges_negative = get_negative_edges_random(
                 subgraph_edges_to_filter=subgraph_edges,
                 all_edges=all_edges,
-                num_negative_edges=len(subgraph_sample_positive),
+                num_negative_edges=10 * len(subgraph_sample_positive),
```

Comment:
should this be in config as well? could it have an effect on our overall training? I assume yes, but that's only an assumption!

Reply:
Yes!